governance/polls/SparkLend Ethereum - Activate Kill Switch Security Module - March 25, 2024.md

```diff
@@ -0,0 +1,51 @@
+---
+title: SparkLend Ethereum - Activate Kill Switch Security Module - March 25, 2024
+summary: Signal your support or opposition to activating the Kill Switch Security Module
+discussion_link: https://forum.makerdao.com/t/mar-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/23918
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-03-25T16:00:00
+end_date: 2024-03-28T16:00:00
+---
+# Poll: SparkLend Ethereum - Activate Kill Switch Security Module - March 25, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, March 25 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the activation of the Kill Switch Security Module on SparkLend Ethereum. The Kill Switch Security Module allows SparkLend Protocol to disable all borrowing across markets in the event of a depeg on key collateral assets.
```

Comment:
:red_circle: **major**

1. The proposal lists defined cutoffs that are proposed for the WBTC/BTC and STETH/ETH pairs. We would include these values in the poll text.
2. Consider linking to the github repo in case voters wish to review the code prior to voting.